Formula/bulk_extractor.rb

```diff
@@ -45,8 +45,19 @@ def install
     # Remove in next version.
     system "autoreconf", "-f"
 
-    system "./configure", "--disable-dependency-tracking",
-                          "--prefix=#{prefix}"
+    # configure cannot find boost libs on Apple Silicon without them being specified
```

Comment:
Does it compile it's own?

Reply:
Nope, but it gave me lots of grief trying to find the boost libs.
It's something about how it detects them.

I can run again and post logs if you are interested.

https://github.com/Homebrew/homebrew-core/blob/1fbc96fe808b40aa9a9a592f940a0a020facc976/Formula/bulk_extractor.rb#L24

Reply:
Try using this config option instead, and remove the `if Hardware::CPU.arm?`:
```
--with-boost=#{Formula["boost"].opt_prefix}
```
This should correctly link to the `boost` formula on all platforms (including Linux, now that the core CI supports it).

If that doesn't work, then try:
```
--with-boost-libdir=#{Formula["boost"].opt_lib}
```
For the record,  `configure --help` shows these Boost-related options:
```
  --with-boost[=ARG]      use Boost library from a standard location
                          (ARG=yes), from the specified location (ARG=<path>),
                          or disable it (ARG=no) [ARG=yes]
  --with-boost-libdir=LIB_DIR
                          Force given directory for boost libraries. Note that
                          this will override library path detection, so use
                          this parameter only if default library detection
                          fails and you know exactly where your boost
                          libraries are located.
  --with-boost-system[=special-lib]
                          use the System library from boost - it is possible
                          to specify a certain library for the linker e.g.
                          --with-boost-system=boost_system-gcc-mt
  --with-boost-filesystem[=special-lib]
                          use the Filesystem library from boost - it is
                          possible to specify a certain library for the linker
                          e.g. --with-boost-filesystem=boost_filesystem-gcc-mt
  --with-boost-iostreams[=special-lib]
                          use the IOStreams library from boost - it is
                          possible to specify a certain library for the linker
                          e.g.
                          --with-boost-iostreams=boost_iostreams-gcc-mt-d-1_33_1
```

Reply:
I’ll give that a shot thanks